### PR TITLE
More doc and changes

### PR DESCRIPTION
**Pull request by thibault-martinez**
## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: documentation/docs/01_installation.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
-# From release
+# Installation
 
-# From source
+## From release
```
Comment (Thoralf-M) [inline documentation/docs/01_installation.md:3]: Do you want to add instructions here?

Comment (thibault-martinez) [reply documentation/docs/01_installation.md:3]: Yes I will, just waiting for the first 1.0 release to happen (later today)

File: documentation/docs/02_account_manager.md [hunk 2]
```diff
@@ -81,22 +79,56 @@ Initialise the wallet with a randomly generated mnemonic and a given node.
...
+$ ./wallet set-node http://localhost:14265/
+```
+
+### `./wallet sync`
+
+Synchronises all accounts.
+
+#### Example(s)
+
+Synchronise all accounts.
```
Comment (Thoralf-M) [inline documentation/docs/02_account_manager.md]: Remove this line?

File: documentation/docs/03_account.md [hunk 3]
```diff
@@ -1,2 +1,218 @@
...
+> Account "main": exit
+```
+
+### `faucet`
+
+#### Parameters
+
+| Name    | Optional  | Default       | Example |
+| ------- | --------- | ------------- | ------- |
+| `` | ✓ | | |
```
Comment (Thoralf-M) [inline documentation/docs/03_account.md:72]: There are optional url and address parameters

File: documentation/docs/03_account.md [hunk 4]
```diff
@@ -1,2 +1,218 @@
...
+> Account "main": help
+```
+
+### `mint-native-token`
+
+#### Parameters
+
+| Name    | Optional  | Default       | Example |
+| ------- | --------- | ------------- | ------- |
+| `` | ✓ | | |
```
Comment (Thoralf-M) [inline documentation/docs/03_account.md:100]: Also missing the parameters

File: documentation/docs/03_account.md (hunk 3 above)
Comment (thibault-martinez) [reply documentation/docs/03_account.md:72]: Yeah I'm doing this page today, just ignore for now

File: documentation/docs/02_account_manager.md (hunk 2 above)
Comment (thibault-martinez) [reply documentation/docs/02_account_manager.md]: Yeah I wanted consistency with all examples but I guess when there is only one I can ignore the description

Comment (thibault-martinez) [reply documentation/docs/02_account_manager.md]: Done

**Review by Thoralf-M — APPROVED** [review]

**Pull request merged**